integration-test/helpers/utils.js

```diff
@@ -0,0 +1,11 @@
+import {constants} from './mocks.js'
+
+const createAvailableInputTypes = (overrides) => {
```

Comment:
can we add some types to these utils? Perhaps you won't need to once the constants have types, but for now we should the `any` trap by being explicit about return values/params if possible :)